optout/src/test/java/gov/cms/ab2d/optout/OptOutProcessorTest.java

```diff
@@ -142,4 +152,27 @@ void getRecordStatusTest() {
         assertEquals(RecordStatus.REJECTED.toString(), optOutProcessing.getRecordStatus());
     }
 
+    @Test
+    void getOptOutResultsTest() throws SQLException {
+        final String OPT_IN_RESULTSET_STRING = "optin";
+        final String OPT_OUT_RESULTSET_STRING = "optout";
+
+        final int OPT_IN_TOTAL = 9;
+        final int OPT_OUT_TOTAL = 7;
+
+        when(resultSet.next()).thenReturn(true).thenReturn(true).thenReturn(false);
```

Comment:
I think I may have one too many `.thenReturn(true)` here. I don't believe the second one is needed?